src/NzbDrone.Core/Indexers/Waffles/WafflesRequestGenerator.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Http;
+using System.Text;
+using NzbDrone.Core.IndexerSearch.Definitions;
+
+namespace NzbDrone.Core.Indexers.Waffles
+{
+    public class WafflesRequestGenerator : IIndexerRequestGenerator
+    {
+        public WafflesSettings Settings { get; set; }
+        
+        public virtual IndexerPageableRequestChain GetRecentRequests()
+        {
+            var pageableRequests = new IndexerPageableRequestChain();
+
+            pageableRequests.Add(GetPagedRequests(null));
+
+            return pageableRequests;
+        }
+
+        [System.Obsolete("Sonarr TV Stuff -- Shouldn't be needed for Lidarr")]
+        public virtual IndexerPageableRequestChain GetSearchRequests(SingleEpisodeSearchCriteria searchCriteria)
```

Comment:
Would it be worth adding an `IAudioIndexerRequestGenerator` so we don't have to implement these? We may at least want to just raise a `NotImplementedException` instead of returning a request change unless that breaks something.